lib/libutee/include/tee_api_defines.h

```diff
@@ -85,8 +91,13 @@
 #define TEE_LOGIN_APPLICATION           0x00000004
 #define TEE_LOGIN_APPLICATION_USER      0x00000005
 #define TEE_LOGIN_APPLICATION_GROUP     0x00000006
-#define TEE_LOGIN_KERNEL                0xA0000000
+#define RSVD_GP_DEFINED_LOGIN_START1    0x00000007
```

Comment:
@jenswi-linaro: 
It is in the spec: ref: Table 4-2: Login Type Constants
Did you ask what RSVD_ means ?
Sure I can update if needed. 
Please provide details how it will be be clear enough for you.